### ISSUE TITLE
Conversely?

### ISSUE THREAD

**Issue by paladin-t** [OneSadCookie/fcaseopen#1]
Any plans of making case-sensitive `fopen` for case-insensitive (Windows) filesystems? Sometimes I prefer to demand relaxed API to follow strict ones, across platforms.

**Comment by OneSadCookie** [comment]
That’s a good idea! I don’t know of such a library. This one was written very specifically to allow an existing, lax app to run on case-sensitive file systems with minimal modification.

**Comment by paladin-t** [comment]
Yup, it would be helpful for some special cases if there was one.
